### PR TITLE
Test topos

### PR DESCRIPTION
**Pull request by TKuh**
I will add the other test for SkeletalFinSets when the next GAP version is released.

I have also kept the sizes of the sets relatively small since otherwise testing might take forever on weaker machines (speaking of experience :D )

### CODE REVIEW

File: tst/CartesianClosedOpposite.tst [hunk 1]
```diff
@@ -0,0 +1,1366 @@
...
+gap> isomorphism_from_coexp_to_d_fsop = Opposite( isomorphism_from_a_to_exp_fs );
+false
+gap> isomorphism_from_coexp_to_d_fsop = Opposite( isomorphism_from_b_to_exp_fs );
+false
+gap> isomorphism_from_coexp_to_d_fsop = Opposite( isomorphism_from_c_to_exp_fs );
+false
+gap> isomorphism_from_coexp_to_d_fsop = Opposite( isomorphism_from_t_to_exp_fs );
+false
+
+gap> STOP_TEST("CartesianClosedOpposite.tst", 1);
```
Comment (mohamed-barakat) [inline tst/CartesianClosedOpposite.tst:1366]: Please add a newline at the end of the file

**Review by mohamed-barakat — CHANGES_REQUESTED** [review]
Could you please also add tests for Cartesian/CocartesianAssociator*?

**Comment by TKuh** [comment]
> Could you please also add tests for Cartesian/CocartesianAssociator*?

Done. I also added tests for the unitors.

File: tst/CartesianClosedOpposite.tst [hunk 2]
```diff
@@ -0,0 +1,1923 @@
+##################################################################################
+# Test the correspondences between the cartesian closed structure of the category
+# of finite sets and the cocartesian coclosed structure of its opposite.
+##################################################################################
+
+gap> START_TEST("CartesianClosedOpposite.tst");
+
+# LoadPackage("FinSets");
```
Comment (zickgraf) [inline tst/CartesianClosedOpposite.tst:8]:
```suggestion
gap> LoadPackage( "FinSetsForCAP", false );
true
```

This way, `FinSetsForCAP` will be loaded when calling the `tst` file directly.

**Review by zickgraf — CHANGES_REQUESTED** [review]
Thanks for your work! Only one small comment, see below.

File: tst/CartesianClosedOpposite.tst (hunk 2 above)
Comment (TKuh) [reply tst/CartesianClosedOpposite.tst:8]: Thank you, I have no idea how that didn't occur to me. I'll also add it in Rows.

**Review by mohamed-barakat — APPROVED** [review]

**Review by zickgraf — APPROVED** [review]

**Pull request merged**